[2022-03-23 20:14:56,655] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-03-02T01:00:00+00:00 [queued]>
[2022-03-23 20:14:56,667] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: EL_dag.extract_task scheduled__2012-03-02T01:00:00+00:00 [queued]>
[2022-03-23 20:14:56,667] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 20:14:56,668] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-03-23 20:14:56,668] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 20:14:56,676] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): extract_task> on 2012-03-02 01:00:00+00:00
[2022-03-23 20:14:56,680] {standard_task_runner.py:52} INFO - Started process 232 to run task
[2022-03-23 20:14:56,683] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EL_dag', 'extract_task', 'scheduled__2012-03-02T01:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/extract_load_dags.py', '--cfg-path', '/tmp/tmpp_a4_n9g', '--error-file', '/tmp/tmphn3zbffi']
[2022-03-23 20:14:56,683] {standard_task_runner.py:77} INFO - Job 105: Subtask extract_task
[2022-03-23 20:14:56,817] {logging_mixin.py:109} INFO - Running <TaskInstance: EL_dag.extract_task scheduled__2012-03-02T01:00:00+00:00 [running]> on host a91ac75a8f43
[2022-03-23 20:14:56,850] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.9/site-packages/airflow/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-23 20:14:56,871] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EL_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2012-03-02T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2012-03-02T01:00:00+00:00
[2022-03-23 20:14:56,871] {logging_mixin.py:109} INFO - 03 2012
[2022-03-23 20:15:07,554] {logging_mixin.py:109} INFO - the response is  200
[2022-03-23 20:15:07,554] {logging_mixin.py:109} INFO - here.........
[2022-03-23 20:15:07,654] {logging_mixin.py:109} INFO -   invoice_line_no                     date  ... sale_liters sale_gallons
0    S04504500029  2012-03-12T00:00:00.000  ...        3.50         0.92
1    S04621000047  2012-03-19T00:00:00.000  ...        7.00         1.85
2    S04759300033  2012-03-27T00:00:00.000  ...        2.25         0.59
3    S04405000019  2012-03-06T00:00:00.000  ...        1.50         0.40
4    S04714700032  2012-03-26T00:00:00.000  ...        1.12         0.30

[5 rows x 24 columns]
[2022-03-23 20:15:07,911] {logging_mixin.py:109} INFO - Data saved..........
[2022-03-23 20:15:07,912] {python.py:175} INFO - Done. Returned value was: None
[2022-03-23 20:15:07,923] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=EL_dag, task_id=extract_task, execution_date=20120302T010000, start_date=20220323T201456, end_date=20220323T201507
[2022-03-23 20:15:07,964] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-23 20:15:08,061] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
